app/models/screening.rb

```diff
@@ -2,5 +2,5 @@ class Screening < ApplicationRecord
   belongs_to :user
   belongs_to :round
 
-  validates :user, uniqueness: { scope: :round }
+  validates :user, uniqueness: { scope: :round_id }
```

Comment:
Just double-checking whether this is correct.

Reply:
Yap. It's a workaround to enable us to use shoulda_matchers without issues to assert this validation. [Source](https://github.com/thoughtbot/shoulda-matchers/issues/814)